src/java.base/share/classes/java/lang/SecurityManager.java

```diff
@@ -80,36 +80,37 @@
  * <p>
  * Environments using a security manager will typically set the security
  * manager at startup. In the JDK implementation, this is done by setting
- * the system property {@code java.security.manager} on the command line to
- * the class name of the security manager. It can also be set to the empty
- * String ("") or the special token "{@code default}" to use the
+ * the system property {@systemProperty java.security.manager} on the command
+ * line to the class name of the security manager. It can also be set to the
+ * empty String ("") or the special token "{@code default}" to use the
  * default {@code java.lang.SecurityManager}. If a class name is specified,
  * it must be {@code java.lang.SecurityManager} or a public subclass and have
  * a public no-arg constructor. The class is loaded by the
  * {@linkplain ClassLoader#getSystemClassLoader() built-in system class loader}
- * if it is not {@code java.lang.SecurityManager}. If the
- * {@code java.security.manager} system property is not set, the default value
- * is {@code null}, which means a security manager will not be set at startup.
+ * if it is not {@code java.lang.SecurityManager}.
  * <p>
  * The Java run-time may also allow, but is not required to allow, the security
  * manager to be set dynamically by invoking the
  * {@link System#setSecurityManager(SecurityManager) setSecurityManager} method.
- * In the JDK implementation, if the Java virtual machine is started with
- * the {@code java.security.manager} system property set to the special token
- * "{@code disallow}" then a security manager will not be set at startup and
- * cannot be set dynamically (the
+ * In the JDK implementation, the default value of the
+ * {@systemProperty java.security.manager} system property, if not set, is
+ * the special token "{@code disallow}". If the Java virtual machine is
+ * started with the {@systemProperty java.security.manager} system property
+ * not set or set to "{@code disallow}" then a security manager will not be
+ * set at startup and cannot be set dynamically (the
  * {@link System#setSecurityManager(SecurityManager) setSecurityManager}
  * method will throw an {@code UnsupportedOperationException}). If the
- * {@code java.security.manager} system property is not set or is set to the
+ * {@systemProperty java.security.manager} system property is set to the
  * special token "{@code allow}", then a security manager will not be set at
```

Comment:
Can/should the `{@systemProperty ...}` tag be used more than once for a given system property? I thought it should be used only once, at the place where the system property is defined. Maybe @jonathan-gibbons can offer some more guidance on this.

Reply:
Good point. I would remove the extra @systemProperty tags on lines 103, 106, and 113. Also, in `System.setSecurityManager` there are 3 @systemProperty java.security.manager tags, I would just keep the first one. (I think it's ok to have more than one, if they are defined in different APIs).

Reply:
New commit pushed. There is only one `@systemProperty` tag now.